packages/react/src/popover/root/PopoverRoot.test.tsx

```diff
@@ -517,8 +517,9 @@ describe('<Popover.Root />', () => {
               </div>,
             );
 
-            const inputInside = screen.getByTestId('input-inside');
-            await act(async () => inputInside.focus());
+            await waitFor(() => {
```

Comment:
A similar things as https://github.com/mui/base-ui/pull/3121/files#r2612924189 seems to be happening here. Wouldn't be surprised if more tests are flaky because of that.